src/css/application.css

```diff
@@ -315,6 +315,20 @@ body {
   border-bottom-left-radius: 0;
 }
 
+.top-bar__project-button-hidden {
```

Comment:
Can just use `u__invisible`?

Reply:
Looks like you did make that change but this class name is still hanging around?